packages/experimental/NativeDatePicker/src/NativeDatePicker.tsx

```diff
@@ -0,0 +1,83 @@
+import { NativeModules } from 'react-native';
+export const NativeDatePicker = NativeModules.MSFDatePickerManager;
+export const { MSFDateTimePickerMode, MSFDateTimePickerDatePickerType, MSFDateTimePickerDateRangePresentation } = NativeDatePicker.getConstants();
+
+type NativeDatePickerMode = keyof typeof MSFDateTimePickerMode;
+type NativeDatePickerType = keyof typeof MSFDateTimePickerDatePickerType;
+type NativeDatePickerDateRangePresentation = keyof typeof MSFDateTimePickerDateRangePresentation;
+
+interface DatePickerParameterObject {
```

Comment:
Nit: Maybe we can we add a comment saying this is coming from the iOS DatePicker? I talked to one person who looked at NativeButton, try to port it to windows, and get confused where some of the very apple-specific props and declarations were coming from.

Reply:
Added in `30c439e`